debian/changelog

```diff
@@ -1,8 +1,19 @@
 datalad (0.12.1-2) UNRELEASED; urgency=medium
 
+  [ Yaroslav Halchenko ]
   * debian/control
     - Recommend datalad-container not datalad-containers
 
+  [ Debian Janitor ]
+  * debian/copyright: use spaces rather than tabs to start continuation
+    lines.
+  * Use secure URI in Homepage field.
+  * Bump debhelper from old 9 to 12.
```

Comment:
Not sure if that would be cool  - might lose some backports, worth checking first